lib/core/enhanceError.js

```diff
@@ -31,10 +31,10 @@ module.exports = function enhanceError(error, config, code, request, response) {
       // Mozilla
       fileName: this.fileName,
       lineNumber: this.lineNumber,
-      columnNumber: this.columnNumber,
+      columnNumber:this.columnNumber,
```

Comment:
format: style mismatch:
```suggestion
      columnNumber: this.columnNumber,
```

`7eb16781` ␣ should be inserted at column 20.